samcli/lib/telemetry/event.py

```diff
@@ -108,6 +111,7 @@ def to_json(self):
             "event_value": self.event_value,
             "thread_id": self.thread_id,
             "time_stamp": self.time_stamp,
+            "exception_name": self.exception_name,
```

Comment:
I assume this is the structure that gets published to Athena, right? Is it safe to publish `None` for the exception type and how does it appear in the database when we need to query it?

Reply:
Yup you're correct that this is what gets published to the database. Publishing `None` is safe, this gets interpreted as `null` in the database (`[{...., exception_name=null, ....}]`), so a query would consist of a null check.